### PR TITLE
Update test1.py

### PR DESCRIPTION
**Pull request by Wildking12**

### CODE REVIEW

**Comment by Wildking12** [comment]
import tensorflow as tf
import numpy as np
from tensorflow import keras
import math




model = tf.keras.Sequential([keras.layers.Dense(units=1, input_shape=[1])])
model.compile(optimizer='sgd', loss='mean_squared_error')

xs = np.array([-1.0, 0.0, 1.0, 2.0, 3.0, 4.0], dtype=float)
ys = np.array([-2.0, 1.0, 4.0, 7.0, 10.0, 13.0], dtype=float)

model.fit(xs, ys, epochs = 100)


a = model.predict([10.0])

if a > 0:
  b = math.floor(a)
print(b)
